executor/yanfd.go

```diff
@@ -131,6 +131,10 @@ func (y *YaNFD) Start() {
 	}
 	dispatch.InitializeFWThreads(fwForDispatch)
 
+	// Initialize FIB table
```

Comment:
This is created too late. Please move it to the beginning of the function. Otherwise the following code will crash:
https://github.com/danningyu/YaNFD/blob/22e55baff7e34dd9de8aa62e4d8ee7bbcec29a2f/executor/yanfd.go#L117-L118

Reply:
Sorry I meant to create a comment for a block of lines 134-137.
I don't understand why GitHub gave this result. Should be a bug.